e2e/pages/shared-lending-platform/unauthenticated-homepage/paperworkReductionAct.spec.ts

```diff
@@ -1,23 +1,22 @@
 import { expect, test } from '@playwright/test';
-import { checkSnapshot } from '../../../utils/snapshotTesting';
+import { expectLinkOpensSameTab } from '../../../utils/openLink';
 import { expectedPaperworkReductionActUrl } from '../../../utils/testFixture.utils';
+import { SelectorLinkText, selectLink } from '../../../utils/verifyLinkTargets';
 
 test('Unauthenticated homepage: Paperwork Reduction Act', async ({ page }) => {
   await test.step('Verify on the Paperwork Reduction Act and link exists', async () => {
```

Comment:
Nice little update here.

NIT: Could we change the title to this to begin with "Verify link" or something just so I can run all these tests as a batch when I search for them?

Reply:
Title updated.